frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/initialize-vm.ts

```diff
@@ -123,7 +121,7 @@ export const initializeVM = (params: CreateVMParams, vm: VMWrapper) => {
   vm.setHostname(
     new CloudInitDataHelper(data).get(CloudInitDataFormKeys.HOSTNAME) ||
       settings[VMSettingsField.HOSTNAME] ||
-      (isTemplate ? VM_TEMPLATE_NAME_PARAMETER : settings[VMSettingsField.NAME]),
+      settings[VMSettingsField.NAME],
```

Comment:
same here, we want to support the NAME parameter for the hostname.
- it might be a bit weird to see your template name as a hostname
- plus the template still could be used somewhere else.

Reply:
Done!